python/lsst/sims/alertsim/catalogs/diaSourceCommons.py

```diff
@@ -1,14 +1,12 @@
 """ DiaSourceCommons """
 from __future__ import absolute_import
 import numpy as np
-import re
+from .dia_transformations import *
```

Comment:
Can you either list the methods/classes being imported or import this with an alias so that we can tell where the imported methods and classes are coming from?  It will make it easier to maintain the code in the future.

Reply:
done